.Rproj.user/F19CAEE1/sources/per/t/EE658456-contents

```diff
@@ -0,0 +1,130 @@
+---
+title: "dataprep options"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{dataprep options}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+<style>
+  body {
+    font-family: 'Montserrat', sans-serif;
+    color: #333;
+    background-color: #f8f9fa;
+  }
+  h1, h2, h3 {
+    color: #007bff;
+  }
+  table {
+    width: 100%;
+    border-collapse: collapse;
+    margin-bottom: 20px;
+  }
+  table, th, td {
+    border: 1px solid #ccc;
+    padding: 10px;
+    text-align: left;
+  }
+  th {
+    background-color: #007bff;
+    color: white;
+  }
+  .section-title {
+    background-color: #007bff;
+    color: white;
+    padding: 10px;
+    margin-top: 20px;
+    margin-bottom: 10px;
+  }
+
+  /* Estilos de la barra de desplazamiento */
+  body::-webkit-scrollbar {
+    width: 10px;
+    background-color: #f8f9fa;
+  }
+
+  body::-webkit-scrollbar-thumb {
+    background-color: #007bff;
+  }
+
+  body::-webkit-scrollbar-thumb:hover {
+    background-color: #0056b3;
+  }
+
+</style>
+
+This document aims to explain the options available in the dataprep category. It will discuss the purpose of each argument, the possible values it can take, and the expected outcome of each argument.
+
+#### agg    
+
+Determines how data is summarized or combined (e.g., sums,    averages, maximums, minimums). It expects a character value   specifying the aggregation function (mean, max, min, etc.).   Its default value is "sum," and its hd_type is Categorical    (Cat).
+
+
+#### agg_na_rm
+
+Indicates whether missing values (NA) should be removed before aggregating data. It expects a value of type logical, which represents boolean values (with possible values being True or False). By default, its value is TRUE, and its hd_type is Check (Chk).
```

Comment:
@NicoSan2304 los datos usados en R son
Numérico (numeric): Representa números reales. Puede ser un número entero o decimal.

Ejemplo: x <- 3.14

Entero (integer): Representa números enteros. Los enteros se distinguen de los números numéricos por el sufijo "L".

Ejemplo: x <- 42L

Carácter (character): Representa texto. Los caracteres se definen utilizando comillas simples o dobles.

Ejemplo: x <- "Hola"

Lógico (logical): Representa valores lógicos verdadero (TRUE) o falso (FALSE).

Ejemplo: x <- TRUE

Factor (factor): Representa variables categóricas o de factor. Los factores se utilizan para almacenar datos categóricos con niveles predefinidos.

Ejemplo: x <- factor(c("Rojo", "Verde", "Azul"))

Fecha (Date): Representa fechas.

Ejemplo: x <- as.Date("2022-01-01")

Fecha y hora (POSIXct y POSIXlt): Representa fechas y horas.

Ejemplo: x <- as.POSIXct("2022-01-01 12:00:00")

Matriz (matrix): Es una estructura de datos bidimensional que contiene elementos del mismo tipo de datos.

Ejemplo: x <- matrix(1:6, nrow = 2, ncol = 3)

Lista (list): Es una estructura de datos que puede contener elementos de diferentes tipos de datos, incluidas otras listas.

Ejemplo: x <- list(1, "Hola", TRUE)

Vector (vector): Es una secuencia ordenada de elementos del mismo tipo de datos.

Ejemplo: x <- c(1, 2, 3, 4)


Entonces en el caso de que el hdtype sea Cat, en R es character,  chk en R es logical,  y asi, puedes poner eg: 
a logical evaluating to TRUE or FALSE (hdtype Chk) y asi con todos los argumentos